src/main/java/Controllers/ExceptionController.java

```diff
@@ -0,0 +1,25 @@
+package Controllers;
+
+import Exceptions.IndexOutOfBounds;
```

Comment:
Je had hier de bestaande exception mogen gebruiken uit java.lang.IndexOutOfBoundsException. Voor java.lang hoef je niks te importeren, dat is het basis package.